apix-impl/src/main/java/eu/xenit/apix/alfresco/workflow/alfresco/AlfrescoProcessInstanceWorkflowConvertor.java

```diff
@@ -93,7 +93,7 @@ private Workflow apply(WorkflowInstance instance, WorkflowTask startTask) {
     }
 
     public void update(String id, WorkflowOrTaskChanges changes) {
-        throw new NotImplementedException();
+        throw new NotImplementedException("Method not yet implemented.");
```

Comment:
No, this is required:
The change is because the actual class has changed. Apache provides this exception in the `lang` and `lang3` libraries respectively. Since `lang` no longer comes with alfresco, `lang3` is already provided in the older versions, and the items we use from `lang` were easily upgradable to their `lang3` versions, I just that.
Though as a consequence, the `lang3` version of `NotImplementedException` does not have a constructor without message.